modules/javafx.controls/src/test/java/test/javafx/scene/control/ListCellStartEditTest.java

```diff
@@ -80,17 +80,21 @@ public void setup() {
     }
 
     @Test
-    public void testStartEdit() {
-        // First test startEdit() without anything set yet.
+    public void testStartEditMustNotThrowNPE() {
+        ListCell<String> listCell = listCellSupplier.get();
```

Comment:
the usual pattern is to create the instances is to do so in setup, not in every single test